tools/boilerplate/boilerplate.generatego.txt

```diff
@@ -0,0 +1,4 @@
+// Copyright Envoy Gateway Authors
```

Comment:
It would be nice if the boilerplate template ended with a blank line so a space exists before the `package` declaration in go files.